src/app/rpmostree-override-builtins.cxx

```diff
@@ -43,6 +44,7 @@ static GOptionEntry option_entries[] = {
   { "reboot", 'r', 0, G_OPTION_ARG_NONE, &opt_reboot, "Initiate a reboot after operation is complete", NULL },
   { "dry-run", 'n', 0, G_OPTION_ARG_NONE, &opt_dry_run, "Exit after printing the transaction", NULL },
   { "lock-finalization", 0, G_OPTION_FLAG_HIDDEN, G_OPTION_ARG_NONE, &opt_lock_finalization, "Prevent automatic deployment finalization on shutdown", NULL },
+  { "cache-only", 'C', 0, G_OPTION_ARG_NONE, &opt_cache_only, "Do not download latest ostree and RPM data", NULL },
```

Comment:
```suggestion
  { "cache-only", 'C', 0, G_OPTION_ARG_NONE, &opt_cache_only, "Do not download RPM data", NULL },
```

Reply:
True...today.  But, I could imagine us changing this in the future...let's say we detect that in order to solve dependencies, we need to update the base ostree.  We don't do this today, but we *could*.

Reply:
We could perhaps make this more generic and say "Only operate on cached data"?

Reply:
I feel like us not doing that automatically (unlike dnf) is an advantage. But definitely open to discuss in the future.

> We could perhaps make this more generic and say "Only operate on cached data"?

Sure, SGTM!